index.html

```diff
@@ -0,0 +1,25 @@
+<html>  
+  <head>
+    <title>Web Calculator</title>
+    <script src="script.js"></script>
+  </head>
```

Comment:
You do not currently link your CSS file in your head. You should do that if you are going to have one. Also, see my comment below for why your CSS is broken.